rl_studio/config.yaml

```diff
@@ -2,18 +2,67 @@ settings:
   output_dir: "./logs/"
   save_model: True
   save_positions: True
-  debug_level: DEBUG
+  debug_level: debug
   telemetry: False
+  telemetry_mask: False
+  plotter_graphic: False
+  #path: /home/docker/RL-Studio #/home/dir/to/RLStudio
+  model_state_name: f1_renault
+  total_episodes: 10_000 #20000
+  training_time: 15 #in hours
+  save_episodes: 10
+  save_every_step: 1_000
+  lap_completed: False
+  my_board: True
+  load_qlearning_pickle_model: False
+  load_qlearning_pickle_file: 1_20210622_1512_actions_set_simple_epsilon_0.99_QTABLE.pkl
+  load_qlearning_model: False
+  load_qlearning_table: train_qlearning_f1_simple_EPISODE_1_20210625-082424-qtable.npy
+  ros_master_uri: '11311'
+  gazebo_master_uri: '11345'
 
 agent:
   f1:
+    agent_name: f1
     camera_params:
-      witdh: 640
+      width: 640
       height: 480
+      center_image: 320 # related to width_image param
+      image_resizing: 100 # percentage resizing image. Number should be multiple of 2 and 5 (5, 10, 20, 25...), otherwise could be unexpected results. Avoid 0
+      num_regions: 16 # divisions in image to get the positions for simplified perception
+    states:
+      state_space: sp1 #sp1 (simplified perception 1 poi), sp3(simplified perception 3 poi), 
+      #sp5(simplified perception 5 poi), spn(simplified perception N poi), image, centers, laser, lidar
+      image: #DQN, DDPG
+        0: [12] # % to get the line from where get the center
+      sp1: 
+        0: [10] #is the line which is going to be processed to get the center
+      sp3:
+        0: [5, 15, 22] #be careful with the bottom of image (480 height x 640 width regular size, but we can take second half: 320 pixels long)
+        # but remember image_resizing: it is easy getting out of image borders
+      sp5: 
+        0: [3, 5, 10, 15, 20]  
+      spn: 
+        0: [10] # Calculating center   
+    rewards:
+      reward_function: discrete # discrete, linear
+      discrete:
+        from_0_to_02: 10 #discrete ad hoc
+        from_02_to_04: 2 #discrete ad hoc
```

Comment:
Move all the comments to a external document that contains the relevant information for the DDPG.

Reply:
Okay, I'll remove the comments.
Comments could also be written in front of each section e.g., at the beginning of settings, agent, actions, environment and algorithm:

#####################
# SETTINGS
# in this section, define commons params
#
############################

###################
# AGENT
# F1 agent
# Camera_params defines...

and so on. I think it is pretty standard